### PR TITLE
add queue metrics

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Adds workqueue metrics. To have insights into queue length & processing time.

I renamed the queue names as they get used as the subsystem for the metrics.
I named them according to the controller they are being used in & the struct they contain.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by xrstf — APPROVED** [review]
LGTM

File: api/pkg/controller/kubeletdnat/kubeletdnat_controller.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func NewController(
 	ctrl := &Controller{
 		client:                   client,
 		nodeLister:               nodeInformer.Lister(),
-		queue:                    workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "nodes"),
+		queue:                    workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "kubetlet_dnat_node"),
```
Comment (thz) [inline api/pkg/controller/kubeletdnat/kubeletdnat_controller.go:66]: typo here "kubetlet" - but i think we should change the name to something aligned with "vpnsidecar" anyways. @mrIncompetent

Comment (mrIncompetent) [reply api/pkg/controller/kubeletdnat/kubeletdnat_controller.go:66]: We could use `vpnsidecar_kubelet_dnat_node`.
But just using `vpnsidecar_node` seems not precise enough, as `vpnsidecar` currently describes the combination of 2 containers - which could contain n controller & queues.

Comment (thz) [reply api/pkg/controller/kubeletdnat/kubeletdnat_controller.go:66]: the `dnat` part (or `dnat_controler`?) makes it precise. the kubelet part is IMO a technical detail (could be used to access anything, not just kubelet-port).
so:
* `vpnsidecar_dnat_controller`

Comment (mrIncompetent) [reply api/pkg/controller/kubeletdnat/kubeletdnat_controller.go:66]: sgtm :)

Comment (mrIncompetent) [reply api/pkg/controller/kubeletdnat/kubeletdnat_controller.go:66]: But please lets add `_node` as sffix, so we follow the existing pattern with name+name of queue resource

Comment (thz) [reply api/pkg/controller/kubeletdnat/kubeletdnat_controller.go:66]: 👍

**Pull request merged**